portal/templates/cluster_node_details.html

```diff
@@ -0,0 +1,90 @@
+
+    <div class="row">
+      <div class="col-lg-12 mx-auto">
+        <ol class="breadcrumb">
+          <li class="breadcrumb-item"><a href="{{url_for('dashboard')}}">Home</a></li>
+          <li class="breadcrumb-item"><a href="{{url_for('list_clusters')}}">Clusters</a></li>
+          <li class="breadcrumb-item"><a :href="'/clusters/'+cluster_name">{[{cluster_name}]}</a></li>
+          <li class="breadcrumb-item active">{[{node_name}]}</li>
+        </ol>
+        <h1>Node: {[{node_name}]}</h1>
+        <hr/>
+        <div class="row">
+          <div class="col-sm-12">
+            <div class="card h-100">
+              <div class="card-header">
+                Monitoring status
+              </div>
+              <div class="card-body">
+                <div class="row">
+                  <div class="col-sm-12 col-md-6 col-lg-6">
+                    <!-- CPU -->
+                    <iframe :src="'https://monitoring.umich-prod.slateci.net/d-solo/6u_Akx7Gk/cluster-monitoring-startout-copy?orgId=1&theme=light&var-DS_PROMETHEUS=SLATE&var-cluster='+cluster_name+'&var-name='+node_name+'&panelId=310'" width="100%" height="330" frameborder="0" style="pointer-events: none"></iframe>
+                  </div>
+                  <div class="col-sm-12 col-md-6 col-lg-6">
+                    <!-- memory -->
+                    <iframe :src="'https://monitoring.umich-prod.slateci.net/d-solo/6u_Akx7Gk/cluster-monitoring-startout-copy?orgId=1&theme=light&var-DS_PROMETHEUS=SLATE&var-cluster='+cluster_name+'&var-name='+node_name+'&panelId=316'" width="100%" height="330" frameborder="0" style="pointer-events: none"></iframe>
+                  </div>
+                </div>
+                <div class="row">
+                  <div class="col-sm-12 col-md-6 col-lg-6">
+                    <!-- network -->
+                    <iframe :src="'https://monitoring.umich-prod.slateci.net/d-solo/6u_Akx7Gk/cluster-monitoring-startout-copy?orgId=1&theme=light&var-DS_PROMETHEUS=SLATE&var-cluster='+cluster_name+'&var-name='+node_name+'&panelId=315'" width="100%" height="330" frameborder="0" style="pointer-events: none"></iframe>
+                  </div>
+                  <div class="col-sm-12 col-md-6 col-lg-6">
+                    <!-- disc -->
+                    <iframe :src="'https://monitoring.umich-prod.slateci.net/d-solo/6u_Akx7Gk/cluster-monitoring-startout-copy?orgId=1&theme=light&var-DS_PROMETHEUS=SLATE&var-cluster='+cluster_name+'&var-name='+node_name+'&panelId=314'" width="100%" height="330" frameborder="0" style="pointer-events: none"></iframe>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<meta id="cluster-data" data-cluster_name="{{cluster_name}}" data-node_name="{{node_name}}">
+<script>
+var cluster = $('#cluster-data').data();
+var cluster_name = cluster.cluster_name;
+var node_name = cluster.node_name;
+
```

Comment:
The 4 lines above this comment is good. You are beginning to set the flask-to-javascript variable translation, which can be used in the VueJS function below (as seen on line 77). However, since you're only including grafana plots here, it doesn't seem like you need VueJS as all. Notice how line 77 starts making an axios call to `/public-clusters-xhr/`, it doesn't seem like you need to make this call on this page. Looks like the only relevant information needed is the `cluster_name` and `node_name`, which you're already passing from python into the flask template.